tests/lib.rs

```diff
@@ -43,6 +43,7 @@ struct Inscribe {
   inscription: String,
   reveal: Txid,
   fees: u64,
+  parent: Option<String>,
```

Comment:
```suggestion
  #[allow(dead_code)]
  parent: Option<String>,
```

Reply:
This is just so that the IDE doesn't show warnings that `inscribe.parent` isn't being used.